accelerator/apis.c

```diff
@@ -6,12 +6,15 @@
  * "LICENSE" at the root of this distribution.
  */
 
+#include <pthread.h>
```

Comment:
Use the HAL functions from entangled: https://github.com/iotaledger/entangled/blob/develop/utils/handles/lock.h